### PR TITLE
Feature/add not blank flag

### PR DESCRIPTION
**Pull request by gonzalog**
Add not-blank flag in order to be able to specify directive to check if the selected date is the last remaining one and, in that case, not to allow user to remove it.

### Screenshot
pickadate

### CODE REVIEW

**Comment by restorandodeploy** [comment]
By analyzing the blame information on this pull request, we identified @gschammah, @ndrisso and @asharbitz to be potential reviewers

File: README.md [hunk 1]
```diff
@@ -131,6 +131,15 @@ The calendar will support selecting multiple dates. NgModel will be set as an ar
 ```html
 <div pickadate multiple></div>
 ```
+
+##### not blank
```
Comment (gschammah) [inline README.md]: we should include here the original attribute. in this case `not-blank`

File: README.md [hunk 2]
```diff
@@ -131,6 +131,15 @@ The calendar will support selecting multiple dates. NgModel will be set as an ar
...
+
+The calendar will not allow user to remove the last remaining date.
+
+```html
+<div pickadate multiple not-blank></div>
```
Comment (gschammah) [inline README.md]: I think that the default behaviour should be `not-blank`, and if the user wants to allow a blank date, it should add the attribute `allow-blank-date` or something like that. what do you think?

**Comment by gschammah** [comment]
great work @gonzalog!

please check my comments

File: src/angular-pickadate.js [hunk 3]
```diff
@@ -380,7 +381,9 @@
             if (index === -1) {
               dateArray.push(dateObj);
             } else {
-              dateArray.splice(index, 1);
+              if (allowBlank || (dateArray.length > 1)){
```
Comment (ndrisso) [inline src/angular-pickadate.js]: why not using an else if?

**Comment by gonzalog** [comment]
@gschammah can you check this once again?

**Comment by gschammah** [comment]
super 👍

**Pull request merged**